packages/shared/lib/services/notification/slack.service.ts

```diff
@@ -40,7 +42,7 @@ interface NotificationPayload {
 interface PostSlackMessageResponse {
     ok: boolean;
     channel: string;
-    ts: string;
```

Comment:
can we keep it and set it ourselves in sendSlackNotification if not present. Then the rest of the code doesn't have to change and we can ensure it is always present